examples/architectureComponentsExample/src/main/java/io/realm/examples/arch/livemodel/LiveRealmObject.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.arch.livemodel;
+
+import android.arch.lifecycle.LiveData;
+import android.support.annotation.MainThread;
+
+import io.realm.ObjectChangeSet;
+import io.realm.RealmModel;
+import io.realm.RealmObject;
+import io.realm.RealmObjectChangeListener;
+
+
+public class LiveRealmObject<T extends RealmModel> extends LiveData<T> {
```

Comment:
Yeah, that makes sense, I just didn't think about the possibility of extending LiveData :smile: 

I based my example on [this](https://github.com/googlesamples/android-architecture-components/blob/178fe541643adb122d2a8925cf61a21950a4611c/BasicSample/app/src/main/java/com/example/android/persistence/viewmodel/ProductListViewModel.java#L44.L56) because it creates the DB and shares the products from its DAO inside the ViewModel, and the whole "open realm, share RealmResults" thing seemed eerily the same thing.